keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepository.kt

```diff
@@ -233,6 +234,16 @@ interface ArtifactRepository : PeriodicallyCheckedRepository<DeliveryArtifact> {
    */
   fun deletePin(deliveryConfig: DeliveryConfig, targetEnvironment: String, reference: String)
 
+  /**
+   * Given artifact details and promotion status, return its last deployed version
```

Comment:
The description of the return here does not match the actual type of the return...

Reply:
true! changed.